### PR TITLE
Rename quad_order to nquads

### PR DESCRIPTION
**Pull request by mayuri-dhote**
#151

### CODE REVIEW

File: psydac/api/ast/fem.py [hunk 1]
```diff
@@ -327,7 +327,7 @@ def __init__(self, expr, terminal_expr, spaces, mapping_space=None, tag=None, ma
             tests               = expr.test_functions
             fields              = expr.fields
             is_broken           = spaces.symbolic_space.is_broken
-            quad_order          = get_quad_order(spaces)
+           nquads          = get_quad_order(spaces)
```
Comment (saidctb) [inline psydac/api/ast/fem.py:330]: you have an indentation problem

File: psydac/api/ast/fem.py [hunk 2]
```diff
@@ -338,7 +338,7 @@ def __init__(self, expr, terminal_expr, spaces, mapping_space=None, tag=None, ma
             trials              = expr.trial_functions
             fields              = expr.fields
             is_broken           = spaces[0].symbolic_space.is_broken
-            quad_order          = get_quad_order(spaces[0])
+           nquads          = get_quad_order(spaces[0])
```
Comment (saidctb) [inline psydac/api/ast/fem.py:341]: same here

File: psydac/api/ast/fem.py [hunk 3]
```diff
@@ -633,7 +633,7 @@ def _create_ast_bilinear_form(terminal_expr, atomic_expr_field,
     g_mats     = BlockStencilMatrixGlobalBasis(trials, tests, pads, m_tests, terminal_expr, l_mats.tag)
     # ...........................................................................................
 
-    if quad_order is not None:
+    ifnquads is not None:
```
Comment (saidctb) [inline psydac/api/ast/fem.py:636]: also here

File: psydac/fem/tensor.py [hunk 4]
```diff
@@ -379,7 +379,7 @@ def spaces( self ):
         return self._spaces
     
     @property
-    def quad_order( self ):
+    defnquads( self ):
```
Comment (saidctb) [inline psydac/fem/tensor.py:382]: this will not work

**Comment by saidctb** [comment]
Thanks @mayuri-dhote for this PR, but the indentation is wrong and needs to be fixed

File: psydac/fem/tensor.py (hunk 4 above)
Comment (mayuri-dhote) [reply psydac/fem/tensor.py:382]: @saidctb what to use then here?

Comment (saidctb) [reply psydac/fem/tensor.py:382]: it should be `def nquads`

**Comment by mayuri-dhote** [comment]
Hi @saidctb can you please review now i have updated the changes.

File: psydac/fem/tensor.py (hunk 4 above)
Comment (saidctb) [reply psydac/fem/tensor.py:382]: you still didn't fix this issue

**Comment by saidctb** [comment]
> Hi @saidctb can you please review now i have updated the changes.

@mayuri-dhote , there is still an issue not fixed

**Comment by mayuri-dhote** [comment]
Hi @saidctb Can you please tell me what I need to fixed.

File: psydac/fem/tensor.py (hunk 4 above)
Comment (saidctb) [reply psydac/fem/tensor.py:382]: @mayuri-dhote this one is still not fixed, you fixed another one and you forgot this

**Comment by mayuri-dhote** [comment]
Hi @saidctb Please review now and merge.

**Comment by saidctb** [comment]
> Hi @saidctb Please review now and merge.

we first need to see if the tests pass

**Comment by saidctb** [comment]
you need to fix the errors shown here https://github.com/pyccel/psydac/runs/5476433750?check_suite_focus=true

**Comment by mayuri-dhote** [comment]
Hi @saidctb How i fixed this error can you give any guidance.

**Comment by yguclu** [comment]
> Hi @saidctb How i fixed this error can you give any guidance.

Hi @mayuri-dhote, I recommend that you just focus on the first failing test and try to fix it:
```
NameError: name 'quad_order' is not defined
```

From what I can see, function `discretize_space` in file `psydac/api/discretization.py` needs to see `quad_order` replaced by `nquads`.

**Comment by mayuri-dhote** [comment]
Hi @saidctb @yguclu please review now

**Comment by yguclu** [comment]
@mayuri-dhote There is the same error in class `DiscreteBilinearForm`:
```
NameError: name 'quad_order' is not defined
```

You can spot these errors by running the unit tests locally. In our [README file](https://github.com/pyccel/psydac#readme) you can find the instructions to install Psydac and run the tests on our machine.

**Pull request closed without merging**